Added code to use swap memory in linux (#5742)
ollama/ollama: docs/faq.md

If you wish to override the `OLLAMA_KEEP_ALIVE` setting, use the `keep_alive` API parameter with the `/api/generate` or `/api/chat` API endpoints.

> - `OLLAMA_LINUX_SWAP` - To use Swap Memory if operating system is Linux.  The default is false.

Reviewer: IMHO more accurate:
```suggestion
- `OLLAMA_LINUX_SWAP` - Account (free) swap as (free) RAM on Linux.  The default is false.
```
